test/test-recipes.js

```diff
@@ -12,7 +12,7 @@ var testUtil = require('./testUtil');
 // Helpers
 // --------------------------------------------------------------------
 
-function toGrammar(recipeString) {
+function toGrammarOrRecipe(recipeString) {
```

Comment:
Maybe just call this `makeRecipe`? Or else it should be `toGrammarOrSemantics` I think.